_translator-files/po/sv-SE/Getting-Started.po

```diff
@@ -234,33 +253,37 @@ msgid ""
 "  <figcaption>The main window when you are connected to a Server</figcaption>\n"
 "</figure>\n"
 msgstr ""
+"<figure>\n"
+"  <img src=\"{% include img/sv-SE-screenshots/main-screen-medium.inc %}\" loading=\"lazy\" alt=\"Screenshot\" style=\"border: 5px solid grey;\">\n"
+"  <figcaption>Huvudfönstret när du är ansluten till en server</figcaption>\n"
+"</figure>\n"
 
 #. type: Plain text
 #: ../wiki/en/Getting-Started.md:84
 #, markdown-text
 msgid "If you don’t want others to hear your audio, click on the “Mute Myself” button which will stop your audio from being sent to other people. They won’t be able to tell you have done this though. But if you see a “mute” icon above a fader, that means they can’t hear you because they've muted your channel in their mix."
-msgstr ""
+msgstr "Om du inte vill att andra ska höra ditt ljud, klicka på knappen “Tysta mig själv” för att stoppa ditt ljud från att skickas till andra. De kommer dock inte att kunna se att du har gjort detta. Men om du ser en “tystad”-ikon ovanför en fader, betyder det att de inte kan höra dig eftersom de har stängt av din kanal i sin blandning."
 
 #. type: Plain text
 #: ../wiki/en/Getting-Started.md:86
 #, markdown-text
 msgid "Note that you can use the Chat facility at any time to message other people while you are connected. The welcome message in the chat may also state some guidelines for use."
-msgstr ""
+msgstr "Observera att du kan använda chat-funktionen när som helst för att skicka meddelanden till andra medan du är ansluten. Välkomstmeddelandet i chatten kan också innehålla vissa riktlinjer för användning."
 
 #. type: Plain text
 #: ../wiki/en/Getting-Started.md:88
 #, markdown-text
 msgid "More information about using Jamulus can be found in the [Software Manual](/wiki/Software-Manual)."
-msgstr ""
+msgstr "Mer information om hur du använder Jamulus hittar du i [Software Manual](/wiki/Software-Manual)."
```

Comment:
Translate text in square brackets.

Reply:
We should change the English here, really... the page title is "Jamulus User Manual" (although "Jamulus" is probably redundant there), which more accurately reflects its content (as opposed to the server manual), which would give "användarmanual" here for the translation.